conda_forge_webservices/update_me.py

```diff
@@ -98,10 +98,15 @@ def main():
             # keep a record around
             pth = os.path.join(clone_dir, "pkg_versions.json")
             with open(pth, "w") as fp:
-                json.dump(final_install, fp)
+                json.dump(final_install, fp, indent=2)
+                fp.write("\n")
             repo.index.add(pth)
 
-            msg_vers = ", ".join(["{}={}".format(k, v) for k, v in to_install.items()])
+            msg_vers = (
+                "Redeploy for package updates:\n\n" +
+                "\n".join(["* `{}={}`".format(k, v) for k, v in to_install.items()])
+            )
+
             repo.index.commit("redeploy for '%s'" % msg_vers)
```

Comment:
So do we just want this?

```suggestion
            repo.index.commit(msg_vers)
```

Reply:
No, the current commit message [redeploy for 'conda=24.3.0'](https://github.com/conda-forge/conda-forge-webservices/commit/bd36e1e228c177bff49efdfeb5603d7757bbb16e) is good enough.

Reply:
However when multiple things are updated like commit ( https://github.com/conda-forge/conda-forge-webservices/commit/8ccbbe9d2beaa6fabeda23a4b2a5b6f72f9e34d9 ), it gets a bit verby

Am happy to simplify, but don't agree with reversion

Reply:
That rarely happens.